packages/build-utils/src/types.ts

```diff
@@ -444,3 +444,5 @@ export type ShouldServe = (
 export type StartDevServer = (
   options: StartDevServerOptions
 ) => Promise<StartDevServerResult>;
+
+export type ProcessEnv = typeof process.env;
```

Comment:
I don't think we should export this.

We need to be really careful about what we export from `@vercel/build-utils` because we can never change it in the future since this package is injected (community builders/runtimes dont use an explicit dependency on it)

Reply:
I just realized we already have `Env` exported so lets use that instead

Reply:
Fixed and it actually made the code cleaner elsewhere.